code/game/objects/items/weaponry.dm

```diff
@@ -470,6 +470,20 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	desc = "A bust of the famous Greek physician Hippocrates of Kos, often referred to as the father of western medicine."
 	icon_state = "hippocratic"
 
+/obj/item/statuebust/attack_self(mob/living/user)
+	user.examinate(src)
+
+/obj/item/statuebust/examine(mob/living/user)
+	. = ..()
+	if(.)
+		return
+	if(!do_after(user, 20, target = src))

```

Comment:
I don't quite understand why you added the cooldown here at all, the common statues don't have one.